lib/sycamore/sycamore/transforms/map.py

```diff
@@ -102,20 +122,12 @@ def __init__(
         f_constructor_kwargs: Optional[dict[str, Any]] = None,
         **resource_args
     ):
-        super().__init__(child, **resource_args)
-        self._f = f
-        self._f_args = f_args
-        self._f_kwargs = f_kwargs
-        self._f_constructor_args = f_constructor_args
-        self._f_constructor_kwargs = f_constructor_kwargs
-
-    def execute(self) -> "Dataset":
-        input_dataset = self.child().execute()
-        if isinstance(self._f, type):
-            ray_callable = generate_map_batch_class(
-                self._f, self._f_args, self._f_kwargs, self._f_constructor_args, self._f_constructor_kwargs
-            )
-            return input_dataset.map_batches(ray_callable, compute=ActorPoolStrategy(size=1), **self.resource_args)
-        else:
-            ray_callable = generate_map_batch_function(self._f)
-            return input_dataset.map_batches(ray_callable, **self.resource_args)
```

Comment:
where did this go?

Reply:
All the interaction with ray happens in BaseMapTransform.

Reply:
The same as the code here?
https://github.com/aryn-ai/sycamore/blob/7c7d3ce4aa48bd1543c70af20a9056f87c52e252/lib/sycamore/sycamore/transforms/base.py#L59

Reply:
yeah, that! where's that thing you linked at me right there ;)
I guess I was blinded by the flashy _process_ray method